src/test/java/teammates/test/cases/BaseTestCaseWithBackDoorApiAccess.java

```diff
@@ -10,26 +10,55 @@
 import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
 import teammates.common.datatransfer.attributes.InstructorAttributes;
 import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.util.Const;
 import teammates.test.driver.BackDoor;
+import teammates.test.driver.retry.RetryManager;
+import teammates.test.driver.retry.RetryableTaskReturns;
 
 /**
  * Base class for all test cases which are allowed to access the Datastore via {@link BackDoor}.
  */
 public abstract class BaseTestCaseWithBackDoorApiAccess extends BaseTestCaseWithDatastoreAccess {
 
+    protected AccountAttributes getAccount(String googleId) {
```

Comment:
> BTW, is it necessary to write a new method protected AccountAttributes getAccount(String googleId)?

In the tests, we hardly deal with full Attributes objects - most of the time, we deal with fields. I find it pretty counter-intuitive to have to construct an entire Attributes object just to use the various methods. To use a method that takes in an Attributes object, I'd need to open the method code to determine which fields of the object it is using, then construct an Attributes object with those fields populated just to pass the data over, only for the method to deconstruct the object again to get the correct fields.

There is no easy way to create an AccountAttributes object when only the account Google ID is known, as there is no AccountAttributes constructor which takes in just a Google ID.